pages/index.js

```diff
@@ -6,7 +6,11 @@ export default function HomePage({
   shoppingItems,
   onCreateItem,
   onDeleteItem,
+  handleTogglePurchased,
 }) {
+  const purchasedItems = shoppingItems.filter((item) => item.isPurchased);
+  const notPurchasedItems = shoppingItems.filter((item) => !item.isPurchased);
```

Comment:
Tiny naming suggestion:

```suggestion
  const unpurchasedItems = shoppingItems.filter((item) => !item.isPurchased);
```